homework/Bladyko Danila/HomeWork_03/by/jrr/src/by/jrr/service/UserLoginService.java

```diff
@@ -0,0 +1,18 @@
+package by.jrr.service;
+
+import by.jrr.main_user.User;
+
+public class UserLoginService {
+    public boolean login(User user, String password) {
+        if (user.password.equals(password) && !user.blocked) {
+            user.loginAttemptsBeforeBlocking++;

```

Comment:
this will increase counter with every successful login. 
[ ] - Use dropCounter() instead